product_attribute_priority/views/product_view.xml

```diff
@@ -0,0 +1,18 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<openerp>
+    <data>
+
+        <record model="ir.ui.view"
+            id="product_attribute_priority_tree_view">
```

Comment:
You may reuse the same id than the original if you want. Whatever module name prefix the id string finally